src/pdn/test/widthtable.py

```diff
@@ -0,0 +1,24 @@
+from openroad import Design, Tech
+import pdn_aux
+import helpers
+
+tech = Tech()
+tech.readLef("asap7_vias/asap7_tech_1x_noviarules.lef")
+tech.readLef("asap7_vias/asap7sc7p5t_27_R_1x.lef")
+
+design = Design(tech)
+design.readDef("asap7_vias/floorplan.def")
+pdngen = design.getPdnGen()
```

Comment:
this isn't used.

Reply:
By that do you mean the entirety of widthtable.py or just the pdngen = design.getPdnGen()? I will assume the latter and update.

Reply:
yes, just the pdngen variable